doc/md/canister-maintenance/cycles.md

```diff
@@ -1,10 +1,12 @@
 # Managing cycles
```

Comment:
I looks like this section wasn't revised yet to use the new <system> capability when adding (and perhaps accepting) cycles. We'll need to revise that eventually though the examples should still compile with a warning.